app/src/main/res/layout/appcoins_radio_button.xml

```diff
@@ -30,13 +30,112 @@
+      />
+  <TextView
+      android:id="@+id/appc_fiat_balance"
+      style="@style/TextAppearance.AppCompat.Body1.Roboto.Regular"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:fontFamily="sans-serif-medium"
+      android:lineSpacingExtra="4sp"
+      app:layout_constraintBottom_toTopOf="@id/balance_eth"
+      app:layout_constraintStart_toStartOf="@id/title"
+      tools:text="31.90 EUR"
+      />
+  <TextView
+      android:id="@+id/appc_text_separator"
+      style="@style/TextAppearance.AppCompat.Body1.Roboto.Medium"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:text="|"
+      android:textSize="12sp"
+      app:layout_constraintBottom_toBottomOf="@id/appc_fiat_balance"
+      app:layout_constraintEnd_toStartOf="@id/appc_balance"
+      app:layout_constraintStart_toEndOf="@id/appc_fiat_balance"
+      app:layout_constraintTop_toTopOf="@id/appc_fiat_balance"
+      />
+
+  <TextView
+      android:id="@+id/appc_balance"
+      style="@style/TextAppearance.AppCompat.Body1.Roboto.Medium"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="8dp"
+      android:fontFamily="sans-serif"
+      android:lineSpacingExtra="4sp"
+      android:textSize="12sp"
+      app:layout_constraintBottom_toBottomOf="@id/appc_text_separator"
+      app:layout_constraintStart_toEndOf="@id/appc_fiat_balance"
+      app:layout_constraintTop_toTopOf="@id/appc_text_separator"
+      tools:text="69.90 APPC"
+      />
+
+  <TextView
+      android:id="@+id/balance_eth"
+      style="@style/TextAppearance.AppCompat.Body1.Roboto.Medium"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginBottom="12dp"
+      android:lineSpacingExtra="6sp"
+      android:text="@string/p2p_send_currency_eth"
+      android:textSize="12sp"
+      app:layout_constraintBottom_toTopOf="@id/appcoins_bonus_layout"
+      app:layout_constraintStart_toStartOf="@id/title"
+      />
+
+  <TextView
+      android:id="@+id/eth_fiat_balance"
+      style="@style/TextAppearance.AppCompat.Body1.Roboto.Regular"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_marginStart="4dp"
+      android:fontFamily="sans-serif-medium"
+      android:lineSpacingExtra="4sp"
+      app:layout_constraintBottom_toBottomOf="@id/balance_eth"
+      app:layout_constraintStart_toEndOf="@id/balance_eth"
+      app:layout_constraintTop_toTopOf="@id/balance_eth"
+      tools:text="31.90 EUR"
+      />
+  <TextView
+      android:id="@+id/eth_text_separator"
+      style="@style/TextAppearance.AppCompat.Body1.Roboto.Medium"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:text="|"
```

Comment:
This attribute should be defined as a untranslatable string in the resources folder